Gemfile

```diff
@@ -3,5 +3,5 @@ source 'https://rubygems.org'
 gem 'rspec'
 gem 'rubocop'
 gem 'destructor'
-gem 'httparty', '0.15.6'
-gem 'pg', '0.19'
```

Comment:
There is [a more recent Dependabot](https://github.com/anynines/unit-tests-utils/pull/65) update that makes a major version bump. Was there a reason for not doing a major bump here? 

It does not seem to be CVE related so we can skip it for now, I'm just curious.

Reply:
If you want to "fix" this version, you could use something like `>~`. I am not sure if Dependabot respects these semantic constraints but it could at least prevent us from doing anything stupid.

Reply:
At least for pg, there are potential breaking changes with the next major update. So to just fix the CVE, only a minor update was made.

I'm not sure about how the dependabot can be restricted to make only minor updates. But there should be a research ticket about the topic. Because this is only a internal component, the risk isn't that high in this case.

Reply:
Ok, if this minor update fixes the CVE then it's all good (for now). I will note down the dependabot-question as well.